.github/workflows/ci.yml

```
@@ -25,20 +25,20 @@ jobs:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
     with:
       envs: |
-        - linux: test-oldestdeps-cov-xdist
+        - linux: test-py39-oldestdeps-cov
           python-version: 3.9
-        - linux: test-xdist
+        - linux: test-py39
           python-version: 3.9
```

```suggestion
        - linux: test-py39
```
Open astronomy is smart enough to determine the python version should be 3.9 from this toxenv name.

---

when enabling this change I got an error https://github.com/spacetelescope/roman_datamodels/pull/187#issuecomment-1561852480

---

I think this is a bug. I'll report it to open astronomy.